### PR TITLE
Ament_lint_auto, and installing rqt_gui resource to package specific destination

### PR DESCRIPTION
**Pull request by brawner**
This addresses comments to several recent PRs.

It removes pytests in favor of ament_lint_auto
It installs rqt_gui resource to a package specific distination.
It also fixes an instance of double quotes.

### CODE REVIEW

File: rqt_py_common/package.xml [hunk 1]
```diff
@@ -33,8 +33,6 @@
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
   <test_depend>ament_cmake_pytest</test_depend>
-  <test_depend>ament_flake8</test_depend>
-  <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
```
Comment (dirk-thomas) [inline rqt_py_common/package.xml]: This test dependency doesn't seem to be needed anymore?

File: rqt_gui/package.xml [hunk 2]
```diff
@@ -16,18 +16,14 @@
 
   <build_depend version_gte="0.3.0">qt_gui</build_depend>
 
-  <exec_depend>ament_cmake_python</exec_depend>
+  <exec_depend>ament_index_python</exec_depend>
   <exec_depend>python_qt_binding</exec_depend>
   <exec_depend>python3-catkin-pkg-modules</exec_depend>
   <exec_depend>qt_gui</exec_depend>
   <exec_depend>rclpy</exec_depend>
-  <exec_depend version_gte="0.3.0">qt_gui</exec_depend>
```
Comment (dirk-thomas) [inline rqt_gui/package.xml]: The version dependency should be kept on the remaining exec dependency.

**Review by dirk-thomas — COMMENTED** [review]
A few of the comments on #148 are still pending (see the non resolved ones on that ticket).

**Review by mlautman — COMMENTED** [review]
+1 after addressing Dirk's feedback and linking a passing test

File: rqt_py_common/package.xml (hunk 1 above)
Comment (brawner) [reply rqt_py_common/package.xml]: Which dependency are you talking about? python3-pytest?

File: rqt_gui/package.xml (hunk 2 above)
Comment (brawner) [reply rqt_gui/package.xml]: Fixed

**Comment by brawner** [comment]
14 tests, 0 errors, 0 failures, 0 skipped
https://ci.ros2.org/job/ci_linux/5509

File: rqt_py_common/package.xml (hunk 1 above)
Comment (dirk-thomas) [reply rqt_py_common/package.xml]: Yes.

Comment (brawner) [reply rqt_py_common/package.xml]: Removed

**Comment by dirk-thomas** [comment]
Please comment on previous comments when they have been resolved (and mark the conversation as resolved). Otherwise it is a back-and-forth trying to figure out what has been addressed and what hasn't.

**Comment by brawner** [comment]
Sorry Dirk, I thought I got them all.

**Comment by mlautman** [comment]
@dirk-thomas LGTM

File: rqt_py_common/src/rqt_py_common/layout_util.py [hunk 3]
```diff
@@ -39,7 +39,7 @@
 
 
 class LayoutUtil(object):
-    _logger = logging.get_logger("LayoutUtil")
```
Comment (dirk-thomas) [inline rqt_py_common/src/rqt_py_common/layout_util.py:42]: I mentioned in the other ticket that this was just one case. The other locations calling the same function still use double quotes.

**Review by dirk-thomas — COMMENTED** [review]
Also https://github.com/ros-visualization/rqt/pull/148#discussion_r232422416 and https://github.com/ros-visualization/rqt/pull/148#discussion_r232422427

**Pull request merged**